perceval/backends/core/slack.py

```diff
@@ -118,7 +119,7 @@ def fetch_items(self, category, **kwargs):
 
         # Minimum value supported by Slack is 0 not 0.0
         if oldest == 0.0:
-            oldest = 0
+            oldest = 0.000000
 
         # Slack does not include on its result the lower limit
```

Comment:
Does it make sense to keep the code below? If dates are rounded to 6 decimals later, this part wouldn't be needed anymore.